reconcile/change_owners/change_types.py

```diff
@@ -380,6 +393,7 @@ def build_change_type_processor(change_type: ChangeTypeV1) -> ChangeTypeProcesso
     """
     ctp = ChangeTypeProcessor(
         name=change_type.name,
+        priority=ChangeTypePriority(change_type.priority),
```

Comment:
this way a PR would fail during PR checks if there is a priority mismatch between q-s and q-r